values.yaml

```
@@ -489,6 +489,13 @@ client:
   # dnsPolicy to use.
   dnsPolicy: null
 
+  # hostNetwork defines whether or not we use host networking instead of hostPort in the event
+  # that a CNI plugin doesnt support hostPort. This has security implications and is not recommended
+  # as doing so gives the consul client unnecessary access to all network traffic on the host.
+  # In most cases, pod network and host network are on different subnets so this should be
```

I feel that subnet is a bit of an IaaS-specific term. I could be wrong though, so this suggestion could be ignored!
```suggestion
  # In most cases, pod network and host network are on different networks so this should be
```